skfda/preprocessing/dim_reduction/projection/_fpca.py

```diff
@@ -27,13 +32,10 @@ class FPCA(ABC, BaseEstimator, TransformerMixin):
     def __init__(self,
                  n_components=3,
                  centering=True,
-                 regularization_parameter=0,
-                 penalty=2):
+                 regularization=None):
         self.n_components = n_components
         self.centering = centering
-        # lambda in the regularization / penalization process
-        self.regularization_parameter = regularization_parameter
-        self.penalty = penalty
+        self.regularization = regularization
```

Comment:
You can pass a regularization parameter to the rregularization object. `compute_penalty_matrix ` still has the parameter because in basis smoothing it is called `smoothing_parameter` and passed to the smoother object, in order to have the same interface as other smoothers.

Reply:
Understood. Then I do not have further questions about this.